filebeat/beater/filebeat.go

```diff
@@ -40,6 +40,11 @@ func (fb *Filebeat) Config(b *beat.Beat) error {
 	// Check if optional config_dir is set to fetch additional prospector config files
 	fb.config.FetchConfigs()
 
+	if len(fb.config.Filebeat.Prospectors) > 0 {
+		fb.config.Filebeat.Inputs = fb.config.Filebeat.Prospectors
```

Comment:
Not allowed. In case we decide to proceed, I will add a check for that. In case both are set, and error is returned.